### ISSUE TITLE
Argentina flag is too small?

### ISSUE THREAD

**Issue by taar1** [blongho/worldCountryData#48]
**Describe the bug**
Argentina flag is too small? Compared to other flags it always appears smaller.

**To Reproduce**
Just use the argentina flag and compare it with others

**Comment by sheinin** [comment]
Yes, many flags are small—NZ, USA, etc. Why do the flag icons of this library have space around images?

**Comment by sheinin** [comment]
Completely agree, many flags appear cropped on the drawable (NZ, AU, etc). I must use `CropTransparent().crop(b)` but it causes ANR. Can the devs address this, please? thanks

**Comment by blongho** [comment]
Hello @taar1 and @sheinin  please provide a fix in a pull request and i will be more than grateful to merge the change. 

Regards,
@blongho
